lib/stats/index.js

```diff
@@ -0,0 +1,10 @@
+exports = module.exports;
```

Comment:
Could be exported like a regular module?

Reply:
Yeah, totally. Will change it.